### ISSUE TITLE
The first button on a page doesn't work in JAWS 10

### ISSUE THREAD

**Issue by NajaWulff** [Notalib/LYT#385]
When opening release #7 in the player with JAWS 10, the first button on all pages page (usually the one in the top left hand corner) doesn't work. It is present on the buttons list, and it is registered when tabbing linearly through the buttons on the page, but it cannot be activated. It works if you click it with the regular cursor, or if you find it with the JAWS search function and hit 'Enter'. 

NB: This issue has not been tested on release #7 with JAWS 12 and 13, so it might in fact also occur in newer JAWS-versions. I will check this after 15:00 today, and return then.
